src/main/java/org/apache/maven/shared/utils/xml/Xpp3DomUtils.java

```diff
@@ -148,15 +148,15 @@ private static boolean isMergeChildren( Xpp3Dom dominant )
     }
 
     /**
-     * @param str The string to be checked.
-     * @return <code>true</code> in case string is empty <code>false</code> otherwise.
+     * @deprecated use <code>str == null || String.isBlank(str)</code> (Java 11+) 
```

Comment:
Why not recommending `https://github.com/apache/maven-shared-utils/blob/f4383c55f77ad4480f146481bf3192bc52c3af30/src/main/java/org/apache/maven/shared/utils/StringUtils.java#L145` as replacement? Or should we rather deprecate that method as well?

Reply:
This is fine as well

Reply:
IMHO

JDK > org,apache.commons > Guava > maven-shared-utils > plexus-utils > random classes  where it doesn't belong

Realistically, this functionality is reinvented so often and so frequently, we'll never get down to a single version. However this case is particularly egregious.